source/_integrations/rki_covid.markdown

```diff
@@ -0,0 +1,55 @@
+---
+title: RKI Covid numbers
+description: Robert-Koch Institut COVID Numbers within Home Assistant.
+ha_category:
+  - Health
+ha_iot_class: Cloud Polling
+ha_release: '2021.12'
+ha_config_flow: true
+ha_codeowners:
+  - '@thebino'
+ha_domain: rki_covid
+ha_platforms:
+  - sensor
+---
+
+This integration allows to visualize the covid numbers provided by the [Robert Koch-Institut](https://www.rki.de). Multiple sensors clustered by districts express detailed information on local regions and areas.
+
+
+
+|Sensor  |Type|Description
+|:-----------|:---|:------------
+|`sensor.NAME_count`| number | indicates the confirmed cases.
```

Comment:
Entity IDs are not guaranteed, in general it is better to describe what sensors are provided instead.